WordPressComKit/MeService.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+import Alamofire
+
+public class MeService {
+    public init() {}
+    
+    public func fetchMe(completion: (Me?, NSError?) -> Void) {
+        Alamofire
+            .request(RequestRouter.Me())
+            .validate()
+            .responseJSON { response in
+                guard response.result.isSuccess else {
+                    completion(nil, response.result.error)
+                    return
+                }
+                
+                let json = response.result.value as? [String: AnyObject]
+                let me = Me()
+                me.ID = json!["ID"] as! Int
```

Comment:
Logged here - https://github.com/Automattic/WordPressComKit/issues/7

Definitely an oversight on my part. Comes from not using Swift 2 a bunch when I first started this. :smile: